assets/course-theme/blocks/lesson-blocks/course-title.block.json

```diff
@@ -11,7 +12,16 @@
     },
     "typography": {
```

Comment:
There are a couple of CSS files for these blocks (`course-title.scss`, `lesson-module.scss` etc), that contain opposing rules, including some `!important` flags. Should those, or parts of them, be also moved into `learning-mode-compat.scss` instead? (Perhaps as part of the opinionated styles PR).

Note that the block styles are currently bundled in `sensei-theme-blocks.scss`, but ideally everything staying there should be theme-agnostic. (For example the progress bar just sets up structure, but course title and lesson module are entirely opinionated styling.)

Reply:
Thanks @yscik , I've moved some styles according to your suggestion [here](https://github.com/Automattic/sensei/pull/6066/commits/483c7e49cd9954076745c3eebbea5d6f92cbb650) in the opinionated styling PR